aws/dynamodb/dynamo.tf

```diff
@@ -41,6 +41,18 @@ resource "aws_dynamodb_table" "vault" {
     type = "S"
   }
 
+  attribute {
+    name = "Retrieved"
+    type = "N"
```

Comment:
Using Number type (0 or 1 to simulate a Boolean) here as Terraform only provides String, Number or Binary data.